package.json

```diff
@@ -57,7 +57,9 @@
     ],
     "coveragePathIgnorePatterns": [
       "/node_modules/",
-      "dist"
+      "dist",
+      "icons/lib/",
```

Comment:
i suggested these edits - `icons/lib` is compiled code, and `slider/src/rc-slider` contains 3rd party code that was copied in so we could add a11y. these were affecting coverage metrics on our own source code.